applications/StructuralMechanicsApplication/test_examples/2DMPCCaseStrMesh.gid/mmread.m

```diff
@@ -0,0 +1,222 @@
+function  [A,rows,cols,entries,rep,field,symm] = mmread(filename)
+%
```

Comment:
Hi Riccardo 

Yes there files are removed in my local commits. Sure I am removing them.